src/core/ext/filters/channel_idle/channel_idle_filter.cc

```diff
@@ -80,123 +79,30 @@ struct MaxAgeConfig {
     return max_connection_age != Duration::Infinity() ||
            max_connection_idle != Duration::Infinity();
   }
-};
-
-/* A random jitter of +/-10% will be added to MAX_CONNECTION_AGE to spread out
-   connection storms. Note that the MAX_CONNECTION_AGE option without jitter
-   would not create connection storms by itself, but if there happened to be a
-   connection storm it could cause it to repeat at a fixed period. */
-MaxAgeConfig GetMaxAgeConfig(ChannelArgs args) {
-  const Duration args_max_age =
-      args.GetDurationFromIntMillis(GRPC_ARG_MAX_CONNECTION_AGE_MS)
-          .value_or(kDefaultMaxConnectionAge);
-  const Duration args_max_idle =
-      args.GetDurationFromIntMillis(GRPC_ARG_MAX_CONNECTION_IDLE_MS)
-          .value_or(kDefaultMaxConnectionIdle);
-  const Duration args_max_age_grace =
-      args.GetDurationFromIntMillis(GRPC_ARG_MAX_CONNECTION_AGE_GRACE_MS)
-          .value_or(kDefaultMaxConnectionAgeGrace);
-  /* generate a random number between 1 - kMaxConnectionAgeJitter and
-   1 + kMaxConnectionAgeJitter */
-  const double multiplier = rand() * kMaxConnectionAgeJitter * 2.0 / RAND_MAX +
-                            1.0 - kMaxConnectionAgeJitter;
-  /* GRPC_MILLIS_INF_FUTURE - 0.5 converts the value to float, so that result
-     will not be cast to int implicitly before the comparison. */
-  return MaxAgeConfig{args_max_age * multiplier, args_max_idle,
-                      args_max_age_grace};
-}
-
-class ChannelIdleFilter : public ChannelFilter {
```

Comment:
Most of this change is lifting this class declaration into the header so that it can be instantiated by the fuzzer.

Reply:
(to do so the MaxAgeConfig had to become a member of MaxAgeFilter, so it got renamed MaxAgeFilter::Config)